README.md

```diff
@@ -24,13 +24,13 @@ This package is available in many languages in the standard packaging formats.
 To use from JavaScript or TypeScript in Node.js, install using either `npm`:
 
 ```bash
-npm install @pulumiverse/pulumi-buildkite
+npm install @pulumiverse/buildkite
```

Comment:
@muhlba91 can you also update the package name in `docs/installation-configuration.md`?

While you are editing that file, can you link the Go module to this URL?
https://pkg.go.dev/github.com/pulumiverse/pulumi-buildkite/sdk/v2/go/buildkite

Reply:
done - can you please check again? :)